api-server/src/server/boot/certificate.js

```diff
@@ -59,6 +56,15 @@ export default function bootCertificate(app) {
   app.use(api);
 }
 
+function verifyCanClaimCert(_req, res) {
+  return res.status(200).json({
```

Comment:
Are we happy with a `200` instead of a `410` (Gone)? @raisedadead

Reply:
Lets do `410`, me likes it.